### PR TITLE
Make requirements clearer

### PR DESCRIPTION
**Pull request by malomarrec**
Some users have tried setting up campaigns without having the right environment.

### CODE REVIEW

File: doc/campaigns/quickstart.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ For more information about campaigns see the ["Campaigns"](index.md) documentati
 
 ## Install the Sourcegraph CLI
 
-In order to create campaigns we need to [install the Sourcegraph CLI](../cli/index.md) (`src`).
+In order to create campaigns we need to [install the Sourcegraph CLI](../cli/index.md) (`src`). Note that there are additional [environment requirements](./references/requirements) for campaigns to work.
```
Comment (mrnugget) [inline doc/campaigns/quickstart.md:15]: How about moving that into line 9, which already mentions "requirement" in another sense. That could be reworked and boldened to make the requirements here stand out more.

Like:

```
Besides a Sourcegraph instance with a some repositories in it, there are **requirements for tooling and the environment in which you create campaigns**, see "[Requirements](./references/requirements.md)") for further information. See ["Quickstart"](../../index.md#quickstart) on how to setup a Sourcegraph instance.
```

Comment (malomarrec) [reply doc/campaigns/quickstart.md]: Much nicer

**Review by chrispine — APPROVED** [review]
LGTM

**Review by eseliger — APPROVED** [review]

File: doc/campaigns/quickstart.md [hunk 2]
```diff
@@ -6,10 +6,12 @@ Get started and create your first [Sourcegraph campaign](index.md) in 10 minutes
 
 In this guide, you'll create a Sourcegraph campaign that appends text to all `README.md` files in all of your repositories.
 
-The only requirement is a Sourcegraph instance with a some repositories in it. See the ["Quickstart"](../../index.md#quickstart) guide on how to setup a Sourcegraph instance.
-
 For more information about campaigns see the ["Campaigns"](index.md) documentation and watch the [campaigns demo video](https://www.youtube.com/watch?v=EfKwKFzOs3E).
 
+## Requirements
+- A Sourcegraph instance with some repositories in it. See ["Quickstart"](../../index.md#quickstart) on how to setup a Sourcegraph instance.
+- A local environment matching [Requirements](./references/requirements.md)") to create Campaigns with the Sourcegraph CLI.
```
Comment (mrnugget) [inline doc/campaigns/quickstart.md:13]: I think the quotes go outside the links, [as per the last point here](https://about.sourcegraph.com/handbook/communication/content_guidelines/style_and_mechanics#instructions-references-and-citations).

The quickstart one didn't have that and the new one had one " :)

```suggestion
- A Sourcegraph instance with some repositories in it. See "[Quickstart](../../index.md#quickstart)" on how to setup a Sourcegraph instance.
- A local environment matching "[Requirements](./references/requirements.md)" to create Campaigns with the Sourcegraph CLI.
```

**Review by mrnugget — APPROVED** [review]

**Review by LawnGnome — APPROVED** [review]
LGTM!

**Pull request merged**